.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,28 +2,25 @@
 Thanks for wanting to fix something on Sequelize - we already love you!
 Please fill in the template below.
 If unsure about something, just do as best as you're able.
-
-If your PR only contains changes to documentation, you may skip the template below.
 -->
 
-### Pull Request Checklist
+## Pull Request Checklist
 
-_Please make sure to review and check all of these items:_
+<!-- Please make sure to review and check all of these items: -->
 
 - [ ] Have you added new tests to prevent regressions?
-- [ ] Does `yarn test` or `yarn test-DIALECT` pass with this change (including linting)?
```

Comment:
This was removed since we test in CI either way right?

Reply:
Yep!